### PR TITLE
MINOR: Update rocksDB dependency to 5.0.1

### PR DESCRIPTION
**Pull request by jozanek**

### CODE REVIEW

**Comment by jozanek** [comment]
4.13.5 is not yet published on central maven repository.

**Comment by ijuma** [comment]
We can only merge this once the jar hits Maven. Maybe you can ask the RocksDB team to publish it to Maven?

**Comment by jozanek** [comment]
Can we retest it after it was pushed to central repo?

**Comment by ijuma** [comment]
@jozi-k, build is in progress. :)

**Comment by guozhangwang** [comment]
test this please

**Comment by guozhangwang** [comment]
test this please

**Comment by guozhangwang** [comment]
@jozi-k It seems RocksDB's latest stable release is `5.0.1`, and they have abandoned `4.13.x`:

http://rocksdb.org/blog/

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ versions += [
   metrics: "2.2.0",
   powermock: "1.6.4",
   reflections: "0.9.10",
-  rocksDB: "4.11.2",
```
Comment (guozhangwang) [inline gradle/dependencies.gradle:41]: should we consider upgrading to latest 5.0.1?

**Comment by jozanek** [comment]
Jenkins, test this please.

**Comment by ijuma** [comment]
Please update the PR title as well.

**Review by ijuma — APPROVED** [review]
LGTM, but will wait for @guozhangwang to confirm that he's happy for this to be in the next release.

**Comment by guozhangwang** [comment]
I'm happy to :) LGTM and merged to trunk.

**Pull request closed without merging**